### PR TITLE
Remove local swagger and re-record

### LINKED ISSUE #16510: Update the autorest.md file to use swagger README instead of json file
related to #16445

### PR DESCRIPTION
**Pull request by christothes**
Fixes #16510

### CODE REVIEW

File: sdk/tables/Azure.Data.Tables/src/autorest.md [hunk 1]
```diff
@@ -8,9 +8,17 @@ Run `dotnet build /t:GenerateCode` to generate code.
...
+    - https://github.com/Azure/azure-rest-api-specs/blob/bda39d9be69b9e838eb41e5b71964a567a627cbc/specification/cosmos-db/data-plane/Microsoft.Tables/preview/2019-02-02/table.json
 namespace: Azure.Data.Tables
 include-csproj: disable
 ```
 
-The direct swagger file reference is temporary until the following issue is addressed https://github.com/Azure/azure-sdk-for-net/issues/13559
+### Fix Response type for QueryEntitiesWithPartitionAndRowKey
+
+``` yaml
+directive:
```
Comment (pakrym) [inline sdk/tables/Azure.Data.Tables/src/autorest.md:19]: Is this tracked anywhere?

**Review by pakrym — APPROVED** [review]

File: sdk/tables/Azure.Data.Tables/src/autorest.md (hunk 1 above)
Comment (christothes) [reply sdk/tables/Azure.Data.Tables/src/autorest.md:19]: Kind of - https://github.com/Azure/azure-rest-api-specs/pull/10194#issuecomment-662709937

I need to follow up on that PR though as it aged out over the break.

Comment (christothes) [reply sdk/tables/Azure.Data.Tables/src/autorest.md:19]: I went ahead and opened a new PR for this change.
https://github.com/Azure/azure-rest-api-specs/pull/12280

**Pull request merged**